xrtpy/response/tests/test_channel.py

```diff
@@ -734,10 +734,6 @@ def test_ccd_gain_right(channel_name):
     channel_filter = Channel(channel_name)
     ccd_gain_right = channel_filter.ccd.ccd_gain_right
 
-    idl_ccd_gain_right_auto = v6_genx_s[_channel_name_to_index_mapping[channel_name]][
-        "CCD"
-    ]["GAIN_R"] * (u.electron / u.DN)
-
```

Comment:
`idl_ccd_gain_right_auto` isn't used now, but I think it was used before we had to correct the value of right gain.